src/main/java/cat/udl/eps/softarch/linkapp/LinkAppApplication.java

```diff
@@ -1,13 +1,40 @@
 package cat.udl.eps.softarch.linkapp;
 
+import cat.udl.eps.softarch.linkapp.domain.Meet;
+import cat.udl.eps.softarch.linkapp.repository.MeetRepository;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.time.ZonedDateTime;
+import java.util.List;
 
 @SpringBootApplication
+@EnableScheduling
 public class LinkAppApplication {
 
-	public static void main(String[] args) {
-		SpringApplication.run(LinkAppApplication.class, args);
-	}
+    public static void main(String[] args) {
+        SpringApplication.run(LinkAppApplication.class, args);
+    }
+
+    @Autowired
+    private MeetRepository meetRepository;
+
+    @Scheduled(cron = "${meet-status-cron}")
+    @Transactional
+    protected void updateMeetStatusCronJob() {
```

Comment:
Implementing this here is quite strange. You should move it to a @Configuration or @Service class.

Reply:
In which package should I put this class?

Reply:
Domain logic is usually kept in Controllers and Services. We are using controllers automatically generated by Spring, through RestRepositories, so our choice is Services. The are implemented in a separate class, annotated with `@Service`, like: https://github.com/UdL-EPS-SoftArch/MyTournamentX-API/blob/master/src/main/java/cat/udl/eps/softarch/mytournamentx/service/MatchWinnerService.java

In your case, the service will include the method being schedules via cron